src/components/OptionsSelector.js

```diff
@@ -226,54 +325,119 @@ class OptionsSelector extends Component {
+                                    <View style={[styles.ph5, styles.pv5, styles.flexGrow1, styles.flexShrink0]}>
+                                        {textInput}
+                                    </View>
+                                </>
+                            ) : (
+                                <>
+                                    <View style={[styles.ph5, styles.pv3]}>
+                                        {textInput}
+                                    </View>
+                                    {optionsList}
+                                </>
+                            )
+                    }
                 </View>
-                {this.props.shouldShowOptions
-                    ? (
-                        <OptionsList
-                            ref={el => this.list = el}
-                            optionHoveredStyle={styles.hoveredComponentBG}
-                            onSelectRow={this.selectRow}
-                            sections={this.props.sections}
-                            focusedIndex={this.state.focusedIndex}
-                            selectedOptions={this.props.selectedOptions}
-                            canSelectMultipleOptions={this.props.canSelectMultipleOptions}
-                            hideSectionHeaders={this.props.hideSectionHeaders}
-                            headerMessage={this.props.headerMessage}
-                            disableFocusOptions={this.props.disableArrowKeysActions}
-                            hideAdditionalOptionStates={this.props.hideAdditionalOptionStates}
-                            forceTextUnreadStyle={this.props.forceTextUnreadStyle}
-                            showTitleTooltip={this.props.showTitleTooltip}
-                        />
-                    )
-                    : <FullScreenLoadingIndicator />}
-            </View>
+                {shouldShowFooter && (
+                    <FixedFooter>
+                        {shouldShowMaxParticipantsMessage && (
+                            <Text style={[styles.textLabelSupporting, styles.textAlignCenter, styles.mt1, styles.mb3]}>
+                                {defaultMaxParticipantsReachedMessage}
+                            </Text>
+                        )}
+                        {!shouldShowMaxParticipantsMessage && this.props.shouldShowOfflineMessage && this.props.network.isOffline && (
+                            <Text style={[styles.formError, styles.pb2]}>
+                                {this.props.translate('session.offlineMessage')}
+                            </Text>
+                        )}
+                        {shouldShowDefaultConfirmButton && (
+                            <Button
+                                success
+                                style={[styles.w100]}
+                                text={defaultConfirmButtonText}
+                                onPress={this.props.onConfirmSelection}
```

Comment:
Bug: Confirm button is pressed when selecting the last particiapant.
1. Split a bill from group chat, and go to the confirm page.
2. Use `arrow up` to scroll to the last participant.
3. Deselect the last participant using `enter`.
4. Re-select the last participant using `enter`.

Expected result: The last participant is selected.
Actual result: The last participant is selected AND the button is pressed.

https://user-images.githubusercontent.com/29673073/171038625-b4fb9c94-467d-441c-b33e-d46884cbedaa.mov

Reply:
Thanks, @rushatgabhane. This is fixed now.